### PR TITLE
🚮 Cleanup cache-service-worker

### PR DESCRIPTION
**Pull request by alanorozco**
Abandoned long ago, but related artifacts were not removed. See #1199

### CODE REVIEW

File: build-system/server/app.js [hunk 1]
```diff
@@ -1479,71 +1475,6 @@ app.get('/dist/amp-inabox-host.(m?js)', (req, _res, next) => {
 /*
  * Start Cache SW LOCALDEV section
  */
```
Comment (alanorozco) [inline build-system/server/app.js:1477]: Can someone confirm that the handler for `/dist/diversions` can also be removed? It's the remaining handler within this block delimited by comments.

Comment (erwinmombay) [reply build-system/server/app.js:1477]: are those routes shared or used with testing?

Comment (erwinmombay) [reply build-system/server/app.js:1477]: cc @rsimha

Comment (rsimha) [reply build-system/server/app.js:1477]: I don't see this being used by any tests. If CI is green after this, that's good enough for me 😃

**Comment by alanorozco** [comment]
Apologies, I accidentally started this PR from my fork's `main` branch. Please follow-up on #33676

**Pull request closed without merging**